datafeeder/datafeeder.properties

```diff
@@ -76,10 +76,44 @@ datafeeder.publishing.backend.geoserver.passwd=${pgsqlPassword}
 datafeeder.publishing.backend.geoserver.[Loose\ bbox]=false
 datafeeder.publishing.backend.geoserver.[Estimated\ extends]=true
 
-logging.level.root=warn
-logging.level.org.springframework=warn
-logging.level.org.hibernate=error
-logging.level.org.georchestra.config.security=debug
-logging.level.org.georchestra.datafeeder=info
-logging.level.org.geoserver.openapi=info
+
+#datafeeder.email.send=true #not used yet
+datafeeder.email.ackTemplate=file:${georchestra.datadir}/datafeeder/templates/analysis-started-email-template.txt
+datafeeder.email.analysisFailedTemplate=file:${georchestra.datadir}/datafeeder/templates/analysis-failed-email-template.txt
+datafeeder.email.publishFailedTemplate=file:${georchestra.datadir}/datafeeder/templates/data-publishing-failed-email-template.txt
+datafeeder.email.publishSuccessTemplate=file:${georchestra.datadir}/datafeeder/templates/data-publishing-succeeded-email-template.txt
+
+administratorEmail=noreply.georchestra.dev@gmail.com
```

Comment:
will be commented out in favor of https://github.com/georchestra/datadir/blob/master/default.properties#L45